Redesign for ESN 20

HidalgoIvan opened the pull request.

Commit:
First commit

Sid0rr0 reviewed (commented):
wrong logo - use the one with border (in ESNCZ20 google drive)

HidalgoIvan committed:
Add string flags

Sid0rr0 committed:
Update logo

fm-mf committed:
Final touches

 - Remove dark theme
 - Remove theme and colour swithcher
 - Update colours and visuals to match other event materials
  - Update texts and logos

fm-mf committed:
Update deployment

fm-mf committed:
Text update
